src/interfaces/breadcrumb-interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface BreadcrumbIF {
+  text: string
+  to?: any
+  href?: string
+}
```

Comment:
Please import and use breadcrumb from bcrs-shared-components.

Reply:
Namerequest UI already has "@bcrs-shared-components/interfaces" as a dependency, so can either:
1. add BreadcrumbIF to the list of external interfaces (in namereqest/src/interfaces/index.ts)
or
2. directly import BreadcrumbIF where you use it (eg, "import { BreadcrumbIF} from '@bcrs-shared-components/interfaces'")

Reply:
Looks good.